### PR TITLE
[4.0] Correct description for custom logging

### PR DESCRIPTION
**Pull request by Fedik**
### Summary of Changes
The description for Log Categories is incorrect.
When the field is empty then custom log is disabled:

https://github.com/joomla/joomla-cms/blob/b1f464ca41320b6117bd78d2d1a17607a86bf137/libraries/src/Application/CMSApplication.php#L1400-L1402

### Testing Instructions

Code review

### Documentation Changes Required

If there a doc about it, then it also need to update.

### CODE REVIEW

**Comment by Fedik** [comment]
This probably for 4.0.1

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 65a29c1d0f2ad22a2ab54887f9f33528264e20e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 65a29c1d0f2ad22a2ab54887f9f33528264e20e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by brianteeman** [comment]
As the very next option is to change the list of categories for custom logging from include to exclude the text here is still not correct.

Perhaps

A comma separated list of log categories to include or exclude

File: administrator/language/en-GB/com_config.ini [hunk 1]
```diff
@@ -89,7 +89,7 @@ COM_CONFIG_FIELD_GZIP_COMPRESSION_LABEL="Gzip Page Compression"
 COM_CONFIG_FIELD_HTMLBODY_LABEL="HTML Body"
 COM_CONFIG_FIELD_LOADBALANCER_ENABLE_DESC="If your site is behind a load balancer or reverse proxy, enable this setting so that IP addresses and other configurations within Joomla automatically take this into account."
 COM_CONFIG_FIELD_LOADBALANCER_ENABLE_LABEL="Behind Load Balancer"
-COM_CONFIG_FIELD_LOG_CATEGORIES_DESC="A comma separated list of log categories to include. Common log categories include but are not limited to: database, databasequery, database-error, deprecated and jerror. If empty, all categories will be shown."
+COM_CONFIG_FIELD_LOG_CATEGORIES_DESC="A comma separated list of log categories to include. Common log categories include but are not limited to: database, databasequery, database-error, deprecated and jerror. If empty, custom logging is disabled."
```
Comment (brianteeman) [inline administrator/language/en-GB/com_config.ini]:
```suggestion
COM_CONFIG_FIELD_LOG_CATEGORIES_DESC="A comma separated list of log categories to include or exclude. Common log categories include but are not limited to: database, databasequery, database-error, deprecated and jerror. If empty, custom logging is disabled."
```

**Comment by richard67** [comment]
Back to pending.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by richard67** [comment]
@Fedik Could you check @brianteeman 's suggestion? Thanks in advance.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on ce1e724a27865b8b1099492dea4f692fa07cf9d3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on ce1e724a27865b8b1099492dea4f692fa07cf9d3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34966">issues.joomla.org/tracker/joomla-cms/34966</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**